src/containers/CodeRevokeModal/CodeRevokeModal.test.jsx

```diff
@@ -101,9 +101,6 @@ describe('CodeRevokeModalWrapper', () => {
     expect(wrapper.find('.modal-body form h3').text()).toEqual('Email Template');
     wrapper.find('.modal-footer .code-revoke-save-btn .btn-primary').hostNodes().simulate('click');
     expect(spy).toHaveBeenCalledWith(couponId, codeRevokeRequestData(1));
-
-    // TODO! uncomment when https://github.com/erikras/redux-form/issues/621 is resolved
-    // expect(wrapper.find('.modal-body form textarea').text()).toEqual(emailTemplate);
```

Comment:
nice catch and cleanup!

Reply:
Yeah, the issue has been resolved but I don't understand what this was trying to test exactly and the test didn't work. Maybe I should try harder to fix it, but...